test/json/entity_list_b3_mynamespace.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
is this a real output from k8s? kubernetes-ro and kubernetes are internal k8s services out of the box, belonging to default namespace.